### ISSUE TITLE
[Longhaul Chaos Test] Failure Daemon

### ISSUE THREAD

**Issue by artursouza** [dapr/dapr#1139]
This service will trigger failures given a configuration.

See child issues for supported failure types.

Parent: #1100 

Implement in C#

**Comment by LMWF** [comment]
It may be possible to use gremlin for many of the faults.  But gremlin is not free above a certain level. 
 https://www.gremlin.com/community/tutorials/how-to-install-and-use-gremlin-with-kubernetes/

**Comment by artursouza** [comment]
I agree. The fault injections can be done via a 3rd party tool, it does not need to be built "in-house".
